normalizers/regex.js

```diff
@@ -254,3 +254,19 @@ export const iosPHDecoder = regexMatchingDecoderMaker(
     /(?<tag>%([0-9.]*[lz]?[@dfsi]|\d+\$[@dfsi]))/g,
     (groups) => ({ t: 'x', v: groups.tag })
 );
+
+// i18next v4 keyNesting format
+//  "keyNesting": "reuse $t(keyDeep.inner)"
+// See: https://www.i18next.com/misc/json-format#i18next-json-v4
+export const i18nextKeyDecoder = regexMatchingDecoderMaker(
```

Comment:
a "key" decoder sounds like a weird name for this. maybe since i18next supports both these `$t()` references and `{{x}}` variables, it'd make sense to have a single "i18nextPHDecoder" that decodes both?

Reply:
Combined both into one decoder